ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -84,28 +83,47 @@ const styles = StyleSheet.create({
   }
 });
 
+const AnimatedScreenContentHeader = Animated.createAnimatedComponent(
+  ScreenContentHeader
+);
+
 const AnimatedTabs = Animated.createAnimatedComponent(Tabs);
-/**
- * A screen that contains all the Tabs related to messages.
- */
-class MessagesHomeScreen extends React.Component<Props, State> {
+
+class MessagesHomeScreen extends React.PureComponent<Props, State> {
   private navListener?: NavigationEventSubscription;
   constructor(props: Props) {
     super(props);
     this.state = {
-      currentTab: 0,
-      hasRefreshedOnceUp: false
+      currentTab: 0
     };
   }
 
-  private animatedScrollPositions: ReadonlyArray<Animated.Value> = [
+  private animatedTabScrollPositions: ReadonlyArray<Animated.Value> = [
     new Animated.Value(0),
     new Animated.Value(0),
     new Animated.Value(0)
   ];
 
-  // tslint:disable-next-line: readonly-array
-  private scollPositions: number[] = [0, 0, 0];
+  /**
+   * The screen header is animated: for each tab, once the y content offset of the
+   * list changes, then the related animatedTabScrollPositions value is updated.
+   * To reproduce a sticky effect common to all the tabs, the animation is based on
+   * the sum of the 3 scroll values.
+   */
+  private sumOfPositions01 = Animated.add(
+    this.animatedTabScrollPositions[0],
+    this.animatedTabScrollPositions[1]
+  );
+  private sumOfPositions = Animated.add(
```

Comment:
```suggestion
  private sumOfPositions = Animated.add(
    Animated.add(
      this.animatedTabScrollPositions[0],
      this.animatedTabScrollPositions[1]
    ),
    this.animatedTabScrollPositions[2]
  );
```
since `this.sumOfPositions01` is never used

Reply:
Actually we have the animatedTabScrollPositions[1] value being always 0. A next story should store the y scroll offset of the message deadline list into it. I suggest to let the animatedTabScrollPositions as a 3 value array (even the second one is still not used)and we will inherit the missing animation management from the proposed code

Reply:
@valentinacala 
Ok. The suggestion is about to avoid the using a method parameter **this.sumOfPositions01** since it is used only to populate **_sumOfPositions_**
My suggestion is to include its creation inside **sumOfPositions**

Reply:
of course, updated within https://github.com/pagopa/io-app/pull/1513/commits/64e1f3530403c2edac82b741a3404274e864353b